tests/test_construct.py

```diff
@@ -0,0 +1,64 @@
+"""Test loading simulation and other objects without validation."""
+import json
+
+import pytest
+import pydantic
+import tidy3d as td
+from tidy3d.log import SetupError, ValidationError
+from .utils import SIM_FULL, clear_tmp
+
+
+def load_sim_dict() -> dict:
+    """Load a sim_dict from file."""
+    return SIM_FULL.copy().dict()
+
+
+def test_construct():
+    """Does the validation occur?"""
+
+    sim_dict = load_sim_dict()
+
+    # add an extra monitor to the sim_dict with identical name
+    sim_dict["monitors"].append(sim_dict["monitors"][0])
+
+    # should error if loading without construct
+    with pytest.raises(SetupError):
+        sim = td.Simulation.parse_obj(sim_dict)
+
+    # should not error if loading with construct
+    sim2 = td.Simulation.construct(sim_dict)
+
+    assert len(sim2.monitors) > 0, "monitor list not added"
+
+    # do something with the loaded sim
+    for monitor in sim2.monitors:
+        monitor.storage_size(10, np.arange(10))
+
+    assert sim == sim2
```

Comment:
`sim` is the one that raises an error if you try to init

Reply:
thanks